src/components/_inputs/InputEnum.vue

```diff
@@ -0,0 +1,12 @@
+<template>
+  <a-form-item :label="schema.title">
+    <a-select :defaultValue="schema.default">
+      <a-select-option v-for="item in schema.enum" :value="item">
```

Comment:
`v-for` sempre com `key`

```suggestion
      <a-select-option
        v-for="item in schema.enum"
        :key="item"
        :value="item"
      >
```